tests/test-admin-deploy-etcmerge-cornercases.sh

```diff
@@ -51,9 +51,21 @@ chmod 700 ${etc}/a/long/dir/forking
 # Symlink to nonexistent path, to ensure we aren't walking symlinks
 ln -s no-such-file ${etc}/a/link-to-no-such-file
 
+# Socket, which should be ignored
+(cd "$etc" && setpriv --pdeathsig SIGTERM -- netcat -lU sock-to-ignore)&
+for x in $(seq 10); do
+  if test -S "${etc}/sock-to-ignore"; then
+    break
+  fi
+  sleep 0.5
+done
+trap 'kill -9 $(jobs -p) || true' EXIT ERR
```

Comment:
This will stomp over any existing traps for those signals. I think we have `libtest_exit_cmds` for this.

Reply:
Bash use of `trap` is full of traps